%% @Introduce  : 
%% @File       : Experience.tex
%% @Author     : ryrl
%% @Email      : ryrl970311@gmail.com
%% @Time       : 2025/01/16 16:44
%% @Description:

\thispagestyle{empty}

I received my master’s degree in Biology and medicine from Soochow University in 2024 under the mentorship of Professor \href{https://ibms.suda.edu.cn/3477/list.htm}{Sidong Xiong} and \href{http://web.suda.edu.cn/rh/}{Professor Hang Ruan}. My research primarily focused on the regulation of A-to-I RNA editing and its implications in autoimmune diseases. Through this work, I gained extensive training in bioinformatics and immunology, providing me with a strong foundation for tackling complex biomedical questions. 
% While my results were not as extensive as I had hoped, the process of exploring scientific questions was deeply motivating. 
During this period, I developed a strong interest in statistics, machine learning, and gene regulation. However, I still feel there is much more to learn, and I intend to pursue further studies in order to make meaningful contributions in these fields. Here is an introduction to my research achievements and experiences: 

\section{Academic Paper}
\begin{enumerate}[label=\arabic*]
	\item \label{circrna} Ruan, H., \textbf{Wang, P.-C.}, \& Han, L. (2022). Characterization of circular RNAs with advanced sequencing technologies in human complex diseases. WIREs RNA, e1759. \url{https://doi.org/10.1002/wrna.1759}. (\textbf{Second author, with the supervisor as the first author; Q1, 6.4})

    \item \label{adar} Weng, S.; Yang, X.; Yu, N.; \textbf{Wang, P.-C.}; Xiong, S.; Ruan, H. Harnessing ADAR-Mediated Site-Specific RNA Editing in Immune-Related Disease: Prediction and Therapeutic Implications. Int. J. Mol.Sci.2024,25,351. \url{https://www.mdpi.com/1422-0067/25/1/351}. (\textbf{Co-author; Q2, 4.9})

    \item \label{nc} Zhou, K., Wei, W., Yang, D. et al. Dual electrical stimulation at spinal-muscular interface reconstructs spinal sensorimotor circuits after spinal cord injury. Nat Commun 15, 619 (2024). \url{https://www.nature.com/articles/s41467-024-44898-9}. (\textbf{Co-author; Q1, 14.9})
\end{enumerate}

\section{Research Projects}
\subsection{Master Thesis}
% \begin{enumerate}[label=(\arabic*)]
\textbf{Characterization and Functional Analysis of A-to-I RNA Editing in Autoimmune Diseases}: This is my independently conducted Master’s thesis, which explores the characteristics and functions of A-to-I RNA editing across six common autoimmune diseases. Using RNA sequencing data from multiple cohorts, the study investigates A-to-I RNA editing levels, ADAR gene expression, and dsRNA-related signaling pathways. It identifies and functionally characterizes A-to-I RNA editing sites, shedding light on their roles in gene regulation, miRNA binding, alternative splicing, and peptide immunogenicity. To further clarify the impact of RNA editing in autoimmunity, Mendelian randomization (MR) analysis is used to distinguish editing events with protective effects from those with pathogenic consequences. In addition, patients are stratified into clusters based on their interferon levels to explore how differences in interferon signaling correlate with RNA editing patterns (Figure \ref{fig:roadmap}). This study ultimately aims to construct a comprehensive atlas of A-to-I RNA editing in autoimmune diseases, offering novel perspectives that may contribute to the identification of potential therapeutic targets.
\begin{figure}[ht]
    \centering
    \includegraphics[width=.92\textwidth]{./figures/A2I_Graphical.pdf}
    \caption{
        Schematic overview of the study design. Bulk RNA-seq data from 2,024 samples covering six autoimmune diseases—Type 1 diabetes (T1D), inflammatory bowel disease (IBD), systemic lupus erythematosus (SLE), rheumatoid arthritis (RA), psoriasis (Pso), and multiple sclerosis (MS)—were retrieved from public repositories. After filtering for high-quality A-to-I RNA editing sites, samples were stratified by interferon levels, and functional analyses were performed to assess recoding, miRNA binding, RBP binding, and alternative splicing. Finally, Mendelian randomization identified causal editing sites, distinguishing those with protective from those with pathogenic effects.
    }
    \label{fig:roadmap}
\end{figure}

\clearpage
\subsection{CoWorks}

\textbf{Dual electrical stimulation at spinal-muscular interface reconstructs spinal sensorimotor circuits after spinal cord injury}\footnote{In this article, I provided suggestions for experiments from a bioinformatics perspective and analyzed the sequencing data, with the main results presented in Figure 9.} (Research Article \ref{nc}): The neural signals produced by varying electrical stimulation parameters lead to characteristic neural circuit responses. However, the characteristics of neural circuits reconstructed by electrical signals remain poorly understood, which greatly limits the application of such electrical neuromodulation techniques for the treatment of spinal cord injury. Here, we develop a dual electrical stimulation system that combines epidural electrical and muscle stimulation to mimic feedforward and feedback electrical signals in spinal sensorimotor circuits. We demonstrate that a stimulus frequency of 10−20Hz under dual stimulation conditions is required for structural and functional reconstruction of spinal sensorimotor circuits, which not only activates genes associated with axonal regeneration of motoneurons, but also improves the excitability of spinal neurons. Overall, the results provide insights into neural signal decoding during spinal sensorimotor circuit reconstruction, suggesting that the combination of epidural electrical and muscle stimulation is a promising method for the treatment of spinal cord injury.

\textbf{Characterization of circular RNAs with advanced sequencing technologies in human complex diseases}\footnote{In this paper, my primary contributions included literature collection and figure creation.} (Review Article \ref{circrna}): Circular RNAs (circRNAs) are one category of non-coding RNAs that do not possess 5′ caps and 3′ free ends. Instead, they are derived in closed circle forms from pre-mRNAs by a non-canonical splicing mechanism named ``back-splicing". CircRNAs were discovered four decades ago, initially called ``scrambled exons''. Compared to linear RNAs, the expression levels of circRNAs are considerably lower, and it is challenging to identify circRNAs specifically. Thus, the biological relevance of circRNAs has been underappreciated until the advancement of next generation sequencing (NGS) technology. The biological insights of circRNAs, such as their tissue-specific expression patterns, biogenesis factors, and functional effects in complex diseases, namely human cancers, have been extensively explored in the last decade. With the invention of the third generation sequencing (TGS) with longer sequencing reads and newly designed strategies to characterize full-length circRNAs, the panorama of circRNAs in human complex diseases could be further unveiled. In this review, we first introduce the history of circular RNA detection. Next, we describe widely adopted NGS-based methods and the recently established TGS-based approaches capable of characterizing circRNAs in full-length. We then summarize data resources and representative circRNA functional studies related to human complex diseases. In the last section, we reviewed computational tools and discuss the potential advantages of utilizing advanced sequencing approaches to a functional interpretation of full-length circRNAs in complex diseases.

\clearpage
\textbf{Harnessing ADAR-Mediated Site-Specific RNA Editing in Immune-Related Disease: Prediction and Therapeutic Implications}\footnote{In this paper, I primarily presented the cutting-edge research status of this field.} (Review Article \ref{adar}): ADAR (Adenosine Deaminases Acting on RNA) proteins are a group of enzymes that play a vital role in RNA editing by converting adenosine to inosine in RNAs. This process is a frequent post-transcriptional event observed in metazoan transcripts. Recent studies indicate widespread dysregulation of ADAR-mediated RNA editing across many immune-related diseases, such as human cancer. We comprehensively review ADARs’ function as pattern recognizers and their capability to contribute to mediating immune-related pathways. We also highlight the potential role of site-specific RNA editing in maintaining homeostasis and its relationship to various diseases, such as human cancers. More importantly, we summarize the latest cutting-edge computational approaches and data resources for predicting and analyzing RNA editing sites. Lastly, we cover the recent advancement in site-directed ADAR editing tool development. This review presents an up-to-date overview of ADAR-mediated RNA editing, how site-specific RNA editing could potentially impact disease pathology, and how they could be harnessed for therapeutic applications.
