%% @Introduce  : 
%% @File       : Introduction.tex
%% @Author     : ryrl
%% @Email      : ryrl970311@gmail.com
%% @Time       : 2025/01/15 13:58
%% @Description: Introduction

\thispagestyle{specialpage}
Recent advancements in high-throughput sequencing technologies, particularly single-cell RNA sequencing (scRNA-seq), have revolutionized our ability to explore gene expression heterogeneity at cellular resolution \cite{nguyen2021comprehensive,dong2024deep}. These techniques enable unprecedented exploration of dynamic biological processes, ranging from cellular differentiation to disease progression \cite{mao2023predicting,lei2024deepgrncs}. A central challenge in this domain, however, lies in deciphering causal molecular interactions from observational omics data \cite{dibaeinia2025interpretable}. This is particularly critical for reconstructing gene regulatory networks (GRNs), in which directed graphs encoding transcription factors' (TFs) regulatory influences on target genes \cite{hernandez2020networks}.

\begin{wrapfigure}[26]{R}[0pt]{0.6\textwidth}
    \vspace{-1em}
    \centering
    \includegraphics[width=.6\textwidth]{./figures/grn.pdf}
    \caption{The major classes of methods for paired single-cell multi-omics GRN inference methods \cite{kim2023gene}. Correlation-based methods, which identify co-variation patterns between molecular features (e.g., TF expression, gene expression, CRE accessibility); Regression approaches, modeling target gene expression as a function of potential predictors like TF expression and/or CRE accessibility; Probabilistic models, aiming to infer the most likely network structure or regulatory interactions based on probability theory; Dynamical systems, using mathematical equations (e.g., differential equations) to model the temporal dynamics of gene expression influenced by regulators and other factors; and Deep learning approaches, employing neural networks to learn complex, potentially non-linear relationships among TFs, CREs, genes, and cellular contexts.}
    \label{fig:grn}
\end{wrapfigure}

Existing computational methods for GRN inference, including correlation-based approaches like SCENIC \cite{aibar2017scenic,bravo2023scenic} that rely on co-expression patterns, and regression models like GRNBoost2 \cite{moerman2019grnboost2} that model target gene expression based on transcription factors, fundamentally depend on association metrics (Figure \ref{fig:grn}). These metrics often conflate causal and correlative relationships \cite{feng2023gene,kim2023gene,mao2023predicting}, leading to the inclusion of spurious edges from indirect regulation \cite{badia2023gene}. This limitation is exacerbated by technical artifacts in scRNA-seq data \cite{wang2024grace}, where introduce high dropout rates and significant noise. While model-based methods \cite{lei2024deepgrncs, feng2023gene}, such as differential equations and probability models (e.g., Bayesian networks) (Figure \ref{fig:grn}), attempt to explicitly model system dynamics or probabilistic dependencies, they often impose restrictive structural or parametric assumptions ill-suited to the complex, nonlinear dynamics of gene regulation and can be computationally expensive for large-scale networks \cite{badia2023gene, feng2023gene}. Similarly, deep learning-based approaches (Figure \ref{fig:grn}), excel at capturing complex, non-linear patterns directly from multi-omics data, they often suffer from poor interpretability, limiting mechanistic biological insights, and typically require large datasets and substantial computational resources for effective training \cite{kim2023gene,shu2021modeling,dong2024deep}. Furthermore, these traditional approaches may overlook other crucial regulatory mechanisms beyond transcript levels, such as chromatin accessibility.

Traditional methods for gene regulatory network (GRN) inference often lack inherent mechanisms for encoding causality \cite{dibaeinia2025interpretable,job2023exploring}. In contrast, Graph Neural Networks (GNNs) can capture the underlying topological patterns of gene regulation through embedding learning, where gene nodes learn feature representations by aggregating information from their network neighbors \cite{otal2024analysis,mao2023predicting,xu2018powerful}. Notably, attention mechanisms, such as the Graph Attention Network (GAT), exhibit a natural connection to causal strength estimation by assigning differential weights to neighboring nodes, allowing the model to prioritize influential regulators \cite{otal2024analysis,wang2024grace,zevcevic2021relating}. Furthermore, graph autoencoders (GAEs), including causal variants, demonstrate the capability to denoise sparse single-cell data by learning latent representations and reconstructing the input, potentially mitigating the impact of technical noise like dropouts \cite{zevcevic2021relating,wang2024grace,mao2023predicting}. Additionally, research suggests that GNNs can function as a type of neural Structural Causal Model (SCM) \cite{zevcevic2021relating}, underscoring their potential for advancing causal inference in GRN analysis.
% Additionally, research indicates that GNNs can be related to or even viewed as a form of neural Structural Causal Model (SCM) \cite{zevcevic2021relating}, a causal inference frameworkï¼Œ which further demonstrates the possibility of using GNN for causal inference

While correlation-based methods can indicate potential regulatory relationships by identifying co-expressed genes \cite{mao2023predicting,kim2023gene}, causal inference offers a more robust approach by aiming to establish direct cause-and-effect relationships, thus revealing which genes actively regulate others \cite{mercatelli2020gene,dibaeinia2025interpretable,wang2024grace}. This move beyond mere association is crucial for overcoming the inherent challenges in deciphering the complexities of GRNs \cite{wang2025diffusion}. Causal inference seeks to disentangle the intricate network of interactions and provide a more accurate and mechanistic understanding of gene regulation \cite{dibaeinia2025interpretable,wang2024grace,otal2024analysis,feng2023gene}. Given that genes within GRNs primarily act as regulators (causes) influencing the expression of other genes (effects), the study of these networks is particularly well-suited for the application and development of new causal inference methodologies. Indeed, there is a growing and significant trend in utilizing causal inference techniques to analyze biological datasets \cite{wang2023dictys,feng2023gene,dong2024deep,du2024causal,dibaeinia2025interpretable} (Table \ref{tab:MethodComparison}), notably to reconstruct gene regulatory networks from gene expression data and other omics modalities, promising deeper insights into cellular processes and disease mechanisms.

Moving beyond the limitations of correlation-based methods that identify co-expression patterns, this research aims to provide a more mechanistic understanding of gene regulation. To achieve this,  this study will couple graph attention mechanisms with causal inference, making edge weights reflect estimated causal strengths rather than mere associations, thereby improving both GRN reconstruction accracy and interpretability.